### PR TITLE
⌨️ type(..., { replace: true}) when entire text value is selected

### LINKED ISSUE #416: ⌨️ Do not clear in type by default ⌨️ "type(..., { replace: true})" when entire text value is selected
"I usually change the values a bit. If I append the filled textfield it replaces the values." from @ZlydeTJ #412

- Do not clear in `type`, remove `options.skipClear`
- Add `type` `options.replace` which selects all before typing. Therefore replacing the text.
- In create, automatically set `options.replace=true` if user clicks to select all, or cmd+a (mac), or ctrl+a (linux/windows) to select all

### PR DESCRIPTION
**Pull request by jperl**
Inline `type(..., { replace: true })` when all the text is selected first

- [x] Port recorder tests to sandbox
- [x] Record selectall for text input
- [x] Remove select all key events
- [x] Add options.replace to buildTypeSteps when selectall precedes typing
- [x] Inline `{ replace: true}` for type steps

Relates to #426
Resolves #416

### CODE REVIEW

**Comment by jperl** [comment]
~~Record selectall for content editables~~ Punting on this for now since it is a corner case and not worth the complexity yet. We would need to [combine two events](https://stackoverflow.com/a/13286950/230462). When someone brings it up we can revisit.

File: packages/build-code/src/StepExpression.ts [hunk 1]
```diff
@@ -20,12 +20,32 @@ export class StepExpression implements Expression {
   private _buildOptions(): string | void {
     const page = this._step.page;
 
+    let options: any = {};
```
Comment (flaurida) [inline packages/build-code/src/StepExpression.ts]: type?

File: packages/build-workflow/src/removeShortcutKeyEvents.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+export const removeShortcutKeyEvents = (
+  eventName: string,
+  events: ElementEvent[]
+) => {
+  const filteredEvents = events.slice(0);
+
+  let key: string;
+  if (eventName === "paste") {
+    key = "v";
+  } else if (eventName === "selectall") {
```
Comment (flaurida) [inline packages/build-workflow/src/removeShortcutKeyEvents.ts]: cut? ✂️

File: packages/build-code/src/StepExpression.ts (hunk 1 above)
Comment (jperl) [reply packages/build-code/src/StepExpression.ts]: fixed 👍

File: packages/build-workflow/src/removeShortcutKeyEvents.ts (hunk 2 above)
Comment (jperl) [reply packages/build-workflow/src/removeShortcutKeyEvents.ts]: What do you mean cut, it's not cut. It's selectall.

Comment (jperl) [reply packages/build-workflow/src/removeShortcutKeyEvents.ts]: We do not currently create code for cut, nor do I think it makes much sense to have cut in a test.

**Pull request merged**